File: mili-api/src/main/java/org/leavesmc/leaves/replay/BukkitRecorderOption.java
package org.leavesmc.leaves.replay;

public class BukkitRecorderOption {
    public String serverName = "Mili";
    public BukkitRecordWeather forceWeather = BukkitRecordWeather.NULL;
    public int forceDayTime = -1;
    public boolean ignoreChat = false;

    public enum BukkitRecordWeather {
        CLEAR, RAIN, THUNDER, NULL
    }
}

File: mili-api/src/main/kotlin/org/leavesmc/leaves/replay/BukkitRecorderOption.kt
package org.leavesmc.leaves.replay

data class BukkitRecorderOption(
    @JvmField var serverName: String = "Mili",
    @JvmField var forceWeather: BukkitRecordWeather = BukkitRecordWeather.NULL,
    @JvmField var forceDayTime: Int = -1,
    @JvmField var ignoreChat: Boolean = false
) {
    enum class BukkitRecordWeather {
        CLEAR,
        RAIN,
        THUNDER,
        NULL
    }
}
